src/ui/customBreakpointsUI.ts

```diff
@@ -3,15 +3,39 @@
  *--------------------------------------------------------*/
 
 import * as vscode from 'vscode';
+import { l10n } from 'vscode';
 import {
   CustomBreakpointId,
   customBreakpoints,
   ICustomBreakpoint,
+  IXHRBreakpoint,
 } from '../adapter/customBreakpoints';
 import { Commands, CustomViews } from '../common/contributionUtils';
 import { EventEmitter } from '../common/events';
 import { DebugSessionTracker } from './debugSessionTracker';
 
+const XHRBreakpointsCategory = 'XHR/Fetch URLs';
```

Comment:
We should localize this string, and inline where it's used (localization is not set up until the extension's activate() is called, so top-level strings like this wouldn't get translated)